public/js/action-list.js

```diff
@@ -106,6 +118,17 @@
                 url = $list.attr('data-icinga-multiselect-url') + '?(' + filters.toArray().join('|') + ')';
             }
 
+            if (! bottomStatusBar.find('.selection-count').length) {
+                bottomStatusBar.prepend('<div class="selection-count"></div>');
+            }
+
+            var $label = $activeItems.data('icinga-multiselect-count-label').replace('%d', $activeItems.length);
```

Comment:
Shouldn't `$activeItems` be `$list` instead? The label attribute is on the list after all.

Reply:
If the attribute is not added in `DetailActions`, then it is so correct because the individual active elements have this attribute and not the list.

Reply:
But it doesn't make any sense that the list items have this label, the label isn't even associated to a particular list item, it only shows a number. Besides, `$activeItems` isn't a list item to begin with, it's a jQuery set of `li`.